sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -0,0 +1,322 @@
+# Azure Text Translation client library for Python
+
+Text translation is a cloud-based REST API feature of the Translator service that uses neural machine translation technology to enable quick and accurate source-to-target text translation in real time across all supported languages.
+
+Use the TextTranslator client library for Python to:
+
+* Return a list of languages supported by Translate, Transliterate, and Dictionary operations.
+
+* Render single source-language text to multiple target-language texts with a single request.
+
+* Convert text of a source language in letters of a different script.
+
+* Return equivalent words for the source term in the target language.
+
+* Return grammatical structure and context examples for the source term and target term pair.
+
+[Source code][python-dt-src]
+| [Package (PyPI)][python-dt-pypi]
+| [Package (Conda)](https://anaconda.org/microsoft/azure-ai-translation-document/)
```

Comment:
I guess we don't have a conda package yet? :)

Reply:
Removed